torch_xla_py/xla_dist.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
I'd prefer not to bolt '3' here, and have is as external requirement.
What is python 4 comes out which is fully compatible with python 3?

Reply:
Done.